record-and-playback/screenshare/scripts/screenshare.yml

```diff
@@ -15,41 +15,5 @@ layout:
       :y: 540
       :width: 240
       :height: 180
-
 formats:
-  - :mimetype: 'video/webm; codecs="vp9, opus"'
```

Comment:
It looks like you've removed the ability to change the generated video formats by editing the config file.
That ability should be kept - people should not have to edit the processing code to change the settings.